docs/DevelopmentSetup.md

```diff
@@ -0,0 +1,36 @@
+# Development
+
+The source code for DIGITS is available on [github](https://github.com/NVIDIA/DIGITS).
+
+To have access to your local machine, you may clone from the github repository with
+```
+git clone https://github.com/NVIDIA/DIGITS.git
+```
+Or you may download the source code as a zip file from the github website.
+
+## Running DIGITS in Development
+
+DIGITS comes with the script to run for a development server.
+To run the development server, use
+```
+./digits-devserver
+```
+
+## Running unit tests for DIGITS
+
+To successfully run all the unit tests, the following plugins have to be installed
+```
+sudo pip install -e ./plugins/data/imageGradients
+sudo pip install -e ./plugins/view/imageGradients
```

Comment:
You don't actually need these installed to run the tests. See https://github.com/NVIDIA/DIGITS/pull/1263.